ecl/hthor/hthor.cpp

```diff
@@ -713,6 +713,7 @@ void CHThorDiskWriteActivity::publish()
         properties.setPropInt64("@totalCRC", totalCRC);
     }
     properties.setPropInt("@formatCrc", helper.getFormatCrc());
+    properties.setPropBool("restricted", (helper.getFlags() & TDWrestricted)!=0);
```

Comment:
@jakesmith Should this always be set, or should it only set if restricted is true?

Reply:
Yes, I would only bother setting the property if true.
So:
```
if (helper.getFlags() & TDWrestricted)
    properties.setPropBool("restricted", true);
```